### DISCUSSION TITLE
Directory structure

### DISCUSSION THREAD

**Discussion by compata** [appium/appium#19337] [Q&A]
Having played around a while with both Appium, Android Studio, and Waydroid, I've got a mish-mash of files and Appium can't find things which are actually there. I have the following directories: ~/Android/Sdk,  /usr/local/android/platform-tools, ~/android-studio, and ~/AndroidStudioProjects. ANDROID_HOME points to ~/Android. How should I clean up this mess? Is there an online list of everything needed and where it ought to be stored? Installing things as I find instructions has led to this mess; it seems things would work out better if there were a complete list in one place.

**Comment by arthuRHD** [comment]
Hi, have you tried Appium Doctor? Aside from a tool chain checker, it's a straight-forward way to have a standard installation if you follow the recommendations.
